magtogoek/adcp/loader.py

```diff
@@ -82,6 +82,7 @@ def load_adcp_binary(
     bad_pressure: bool = False,
     start_time: str = None,
     time_step: float = None,
+    magnetic_declination_setting: float = None,
```

Comment:
I think that `magnetic_declination_preset` or `magnetc_declination_raw_preset` would be a more accurate term. (preset: set previously).

Reply:
Seems odd to me because I have never seen an ADCP dataset where this setting was intentionally left blank. Setting it before deployment is certainly a very common thing to do, for fixed locations at least. But as long as it is well documented it's an implementation detail. Changed to `magnetic_declination_preset`.